utilities/colorPalettePlugin.ts

```diff
@@ -54,5 +56,7 @@ export const colorNameToHex = {
   'digital-red-xlight': '#F83535',
   white: '#FFFFFF',
   'plum-light': '#734675',
-  'black-50': '#00000080',
+  'black-50': '#00000080', 
+  'gc-black-10': '#F3F3F9',
+  'gc-black': '#17171A',
```

Comment:
@jenbreese I looked at your latest commit. I believe you just need to add 'gc-black' and 'gc-black-10' to `accentBgColors` in the datasource.ts file 😄